sklearn/notebooks/census_prediction/local_train.py

```diff
@@ -0,0 +1,143 @@
+# [START imports]
+import json
+import numpy as np
+import os
+import pandas as pd
+import pickle
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.externals import joblib
+from sklearn.feature_selection import SelectKBest
+from sklearn.pipeline import FeatureUnion
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import LabelBinarizer
+# [END imports]
+
+
+# [START define-input-data]
+# Define the format of your input data, including unused columns.
+# These are the columns from the census data files.
+COLUMNS = (
+    'age',
+    'workclass',
+    'fnlwgt',
+    'education',
+    'education-num',
+    'marital-status',
+    'occupation',
+    'relationship',
+    'race',
+    'sex',
+    'capital-gain',
+    'capital-loss',
+    'hours-per-week',
+    'native-country',
+    'income-level'
+)
+
+# Categorical columns are columns that need to be turned into a numerical value
+# to be used by scikit-learn
+CATEGORICAL_COLUMNS = (
+    'workclass',
+    'education',
+    'marital-status',
+    'occupation',
+    'relationship',
+    'race',
+    'sex',
+    'native-country'
+)
+# [END define-input-data]
+
+
+# [START load-and-convert-data]
+# Load the training census dataset
+with open('./census_data/adult.data', 'r') as train_data:
+    raw_training_data = pd.read_csv(train_data, header=None, names=COLUMNS)
+
+# Remove the column we are trying to predict ('income-level') from our features list
+# Convert the Dataframe to a lists of lists
+train_features = raw_training_data.drop('income-level', axis=1).as_matrix().tolist()
+# Create our training labels list, convert the Dataframe to a lists of lists
+train_labels = (raw_training_data['income-level'] == ' >50K').as_matrix().tolist()
+
+# Load the test census dataset
+with open('./census_data/adult.test', 'r') as test_data:
+    raw_testing_data = pd.read_csv(test_data, names=COLUMNS, skiprows=1)
+# Remove the column we are trying to predict ('income-level') from our features list
+# Convert the Dataframe to a lists of lists
+test_features = raw_testing_data.drop('income-level', axis=1).as_matrix().tolist()
+# Create our training labels list, convert the Dataframe to a lists of lists
+test_labels = (raw_testing_data['income-level'] == ' >50K.').as_matrix().tolist()
+# [END load-and-convert-data]
```

Comment:
From Line 63: # Load the test census dataset to Line 70.

Test_Features need to be moved into the predict.py file. 

+ add `import pandas as pd` to top of file.